src/scss/02_atoms/_jumbotron.scss

```diff
@@ -0,0 +1,44 @@
+$jumbotronConfig: (
+  backgroundColor: color(background, dark),
+  borderRadius: rem(20px),
+  paddingBottom: rem(30px),
+  paddingLeft: rem(30px),
+  paddingRight: rem(30px),
+  paddingTop: rem(25px),
+  breakpoint: (
+    small: (
```

Comment:
Tutti i valori che cambiano al cambiare dei breakpoints vanno definiti nella mappa dei breakpoints.

Reply:
Gli unici valori non inseriti sono quelli riguardanti flex, che però non abbiamo mai incluso in quanto non devono essere modificate altrimenti il componente si rompe.